00_сompilation_process/00_01_compilation/00_01_01_overloads/00_01_01_overloads.cpp

```diff
@@ -0,0 +1,11 @@
+// Show the symbol table than add extern "C" to function
```

Comment:
than -> then (appears in other places)
Consider using "and" instead of then or just defining each point on the separate line without any adverbs.